db/migrate/20201006074131_add_first_last_name_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddFirstLastNameToUser < ActiveRecord::Migration[6.0]
```

Comment:
name ends with plural `Users` as we use table name, you can check other migrations.